GitSearch/GitSearch/Base.lproj/LaunchScreen.storyboard

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file is used by application - it is a [launch screen](https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/launch-screen/) shown during app launch. Its probably a mistake to delete this file (especially if corresponding key [UILaunchStoryboardName](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/iPhoneOSKeys.html#//apple_ref/doc/uid/TP40009252-SW40) from ```Info.plist``` file is still present).

Reply:
I saw another storyboard file and decide that I need to delete it too.
OMG! How to get back it if I made this action at the start and then made all of this branches

Reply:
git checkout --GitSearch/GitSearch/Base.lproj/LaunchScreen.storyboard
something like that or it's to late?

Reply:
and why it still work, and there is no crash of application - if you said this is important file

Reply:
It works because its not crucial component, just piece of UI to show when your app starts till the moment it will be fully loaded and able to present its own UI.